src/edit-plan-form.ts

```diff
@@ -3,26 +3,63 @@ import { customElement, property } from 'lit/decorators.js';
 
 import type { MonthlyPlan } from './models/plan';
 import './form-sections/cancel';
+import './form-sections/amount';
+import type { MGCEditPlanAmount } from './form-sections/amount';
 
 @customElement('ia-mgc-edit-plan')
 export class IauxEditPlanForm extends LitElement {
   @property({ type: Object }) plan?: MonthlyPlan;
 
   @property({ type: Object }) cancelPlanHandler?: (plan: MonthlyPlan) => void;
 
+  @property({ type: Object }) updateAmountHandler?: (
+    plan: MonthlyPlan,
+    amountUpdates: {
+      amount: number;
+      baseAmount: number;
+      coverFees: boolean;
+      feeCovered: number;
+    }
+  ) => void;
+
+  createRenderRoot() {
+    return this;
+  }
+
+  amountUpdates(status: 'success' | 'fail') {
+    const amountForm = this.querySelector(
+      'ia-mgc-edit-plan-amount'
+    ) as MGCEditPlanAmount;
+    amountForm!.amountUpdated(status);
+  }
+
   render() {
     return html`
       <section class="mgc-edit-plan">
-        <hr />
-        <ia-mgc-cancel-plan
+        <ia-mgc-edit-plan-amount
           .plan=${this.plan}
-          @cancelPlan=${() => {
+          @updateAmount=${(e: CustomEvent) => {
+            const { amount, baseAmount, coverFees, feeCovered } = e.detail;
             if (this.plan) {
-              this.cancelPlanHandler?.(this.plan);
+              this.updateAmountHandler?.(this.plan, {
+                amount,
+                baseAmount,
+                coverFees,
+                feeCovered,
+              });
             }
           }}
-        ></ia-mgc-cancel-plan>
+        ></ia-mgc-edit-plan-amount>
+        <hr />
+        <p class="email-edit-plan">
+          Need to update your plan further? Please email us at
```

Comment:
SUGGESTION: Should be using localization function msg() everywhere.

Reply:
thanks. i will add when we move to offshoot as it has the convention there